xml/containers-orchestration.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+    [
+<!ENTITY % entities SYSTEM "entity-decl.ent">
+%entities;
+]>
+<chapter xml:id="cha-orchestration" xml:lang="en"
+    xmlns="http://docbook.org/ns/docbook" version="5.0"
+    xmlns:xi="http://www.w3.org/2001/XInclude"
+    xmlns:xlink="http://www.w3.org/1999/xlink">
+ <info>
+  <title>Container Orchestration</title>
+ </info>
+ <sect1 xml:id="sec-pod-deployment-with-podman">
+  <title>Pod Deployment with Podman</title>
+
+  <para>
+   In addition to building and managing images, &podman; makes it possible to
+   work with pods. A pod is a group of one or more containers, with shared
+   storage and network resources. A pod usually encapsulates an application
+   composed of multiple containers into a single unit.
+  </para>
+
+  <para>
+   The <command>podman pod</command> can be used to create, delete, query, and
+   inspect pods. To create a new pod, run the <command>podman pod create --name
+   <replaceable>POD_NAME</replaceable></command> command. This creates a pod
+   with a random name. To list the existing pods, use the <command>podman pod
```

Comment:
If you run `podman pod create --name foo` it won't create a pod with a random name :)